eslintrc.json

```diff
@@ -9,12 +9,13 @@
 
   "settings": {
     "react": {
-      "version": "detect"
+      "version": "latest"
```

Comment:
I'd assume `detect` is correct as we can't assume the developer is always on the latest React release?

Reply:
Please revert this